src/main/java/ru/hogwarts/school/model/Faculty.java

```diff
@@ -1,9 +1,14 @@
 package ru.hogwarts.school.model;
 
-import java.util.Objects;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
 
+import java.util.Objects;
+@Entity
 public class Faculty {
-
+    @Id
+    @GeneratedValue
```

Comment:
Напиши вот так, должно заработать
```suggestion
@GeneratedValue(strategy = GenerationType.IDENTITY)
```